lib/TPP/DefaultTppPasses.cpp

```diff
@@ -233,14 +230,14 @@ struct TppMapping : public tpp::impl::TppMappingBase<TppMapping>,
 };
 
 // Convert all matching operations to TPP.
+// TODO: We can remove it now. It has a single pass inside.
```

Comment:
I will address this TODO in a later PR if is ok for you.

Reply:
Fair enough, we can cleanup pipeline and other passes after.